lib/templates.php

```diff
@@ -258,3 +259,15 @@ function filter_rest_wp_template_query( $args, $request ) {
 }
 add_filter( 'rest_wp_template_query', 'filter_rest_wp_template_query', 99, 2 );
 
+/**
+ * Filter the default template types to temporarily remove 'embed',
+ * because it is not a regular template type.
+ *
+ * @param array $default_template_types The array of template types definitions.
+ * @return array Filtered $default_template_types.
+ */
+function gutenberg_filter_default_template_types( $default_template_types ) {
+	unset( $default_template_types['embed'] );
```

Comment:
I know I mentioned this in the previous review, but my opinion is still that we just shouldn't add `embed` to the template type list in the first place if it's somehow different from everything else